sssom/io.py

```diff
@@ -107,12 +114,11 @@ def validate_file(input_path: str) -> bool:
     :param input_path: The path to the input file in one of the legal formats, eg obographs, aligmentapi-xml
     :returns: True if valid SSSOM, false otherwise.
     """
-    try:
```

Comment:
extend validate_file() method to parameter to 

```
validate_file(input_path: str, validation_types = DEFAULT_VALIDATION_TYPES: list[str]) -> bool
```

in constants.py (or wherever our config resides):

```
class SchemaValidationType(Enum):
    JsonSchema = 1
    Shacl = 2
    PrefixMapCompleteness = 3

DEFAULT_VALIDATION_TYPES = [ SchemaValidationType.JsonSchema, SchemaValidationType.PrefixMapCompleteness ]
```